dsp/window/window_test.go

```diff
@@ -5,82 +5,111 @@
 package window
 
 import (
+	"fmt"
 	"testing"
 
 	"gonum.org/v1/gonum/floats"
 )
 
 var windowTests = []struct {
-	name   string
-	fn     func([]float64) []float64
-	want   []float64
-	winLen int
+	name    string
+	fn      func([]float64) []float64
+	fnCmplx func([]complex128) []complex128
+	winLen  int
```

Comment:
I think `winLen` can be taken from the length of `want`.

Reply:
Done.